lib/src/lifecycle_module.dart

```diff
@@ -297,6 +303,13 @@ abstract class LifecycleModule {
     return completer.future;
   }
 
+  /// Aliased to [unload].
+  ///
+  /// Deprecated: Use the method [unload] instead.
+  @deprecated
+  @override
+  Future<Null> dispose() => unload();
```

Comment:
@maxwellpeterson-wf would you like me to add it in addition to the other attributes?